### PR TITLE
Fix CIM100 CGMES import regression issue on synchronous machine

### PR DESCRIPTION
**Pull request by rygx**
**Please check if the PR fulfills these requirements**

- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem?**
Not, but we discovered the issue in slack channel.

**What kind of change does this PR introduce?**
Bug fix - CIM 100 condenser import performance regression.

**What is the current behavior?**
Importing CGMES3 data takes a long time, and most of the time is spent on importing synchronous machines. Activsg70k dataset import takes more than 1 hour.

**What is the new behavior (if this is a feature change)?**
Activsg70k import now takes 1 minute.

**Does this PR introduce a breaking change or deprecate an API?**
- [ ] Yes
- [x] No

**What changes might users need to make in their application due to this PR? (migration steps)**

No actions needed.

**Other information**:

### CODE REVIEW

File: cgmes/cgmes-model/src/main/resources/CIM100.sparql [hunk 1]
```diff
@@ -63,7 +63,7 @@ WHERE {
         cim:SynchronousMachineKind.motor
         cim:SynchronousMachineKind.generatorOrMotor
         cim:SynchronousMachineKind.motorOrCondenser
-        cim:SynchronousMachineKind.generatorOrMotorOrCondenser
+        cim:SynchronousMachineKind.generatorOrCondenserOrMotor
```
Comment (jeandemanged) [inline cgmes/cgmes-model/src/main/resources/CIM100.sparql:66]: damn, good catch. Needs also be applied to CIM16 and CIM14 ...

**Comment by zamarrenolm** [comment]
For CIM16 it was already fixed in #2933. Sorry, In that change I missed the CIM100 variant.

**Comment by zamarrenolm** [comment]
I will prepare the fix for CIM14 in a separate PR

**Review by zamarrenolm — APPROVED** [review]

**Review by olperr1 — APPROVED** [review]

**Comment by olperr1** [comment]
@rygx: Thank you for your contribution!

**Pull request merged**